docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -102,9 +102,18 @@ children:
     url: Repositories.html
     output: 'web, pdf'
 
+  - title: 'Relations'
+    url: Relations.html
+    output: 'web, pdf'
+
   - title: 'Decorators'
     url: Decorators.html
     output: 'web, pdf'
+    children:
```

Comment:
Does this work as intended? Shouldn't it be under `Relations`?

Reply:
Good catch. Fixed :)